packaging/dbscripts/upgrade/04_05_0100_fix_04_04_0050_set_to_not_null.sql

```diff
@@ -1,2 +1,21 @@
+-- Make sure that forgotten options from ancient releases have default value
+UPDATE vdc_options
+  SET default_value = option_value
+  WHERE
+    default_value IS NULL
+    AND option_value IS NOT NULL;
+
+-- If there are still some crappy options, let's set default value to empty string
```

Comment:
@mwperina wouldn't it be better to remove such values instead?

Reply:
Yes, what is the point to preserve those entries ?

Reply:
I don't think there is any reasonable way how to find out which options should be there and which not. The only reliable is to have a new 4.5 installation and then look at vdc_options table content. But that can't be done before the upgrade. But feel free to suggest a better way, maybe I missed something

Reply:
what I had in mind is:
drop from vdc_options where default_value is null and option_value is null

Reply:
It seems too dangerous to me, we have no way to verify if we haven't removed anything important. It's safer to leave those items there, it's just a penalty as we haven't noticed for several years that handling NULL/NOT NULL in fn_db_change_column_type doesn't work properly

Reply:
+1 for being conservative. if any other change is needed, then it needs to happen on master branch first anyway